src/figuresClasses.ts

```diff
@@ -2,10 +1,102 @@
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle';
+
+  color: 'red' | 'green' | 'blue';
+
+  private a: number;
+
+  private b: number;
+
+  private c: number;
+
+  constructor(
+    color: 'red' | 'green' | 'blue',
+    a: number,
+    b: number,
+    c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Sides must be greater than 0');
```

Comment:
[CODE STYLE] - If you throw an error try to provide its detailed description in error message. The error message 'Sides must be greater than 0' could be more descriptive, for example, 'Triangle side lengths must be greater than 0'.